spec/api/source_rpm_changelog_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'Changelog' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.